mpi4py.spec

```diff
@@ -81,32 +78,33 @@ BuildRequires:  python2-Cython >= 0.22
 %if 0%{?with_python3}
 BuildRequires:  python3-rpm-macros
 BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
 BuildRequires:  python%{python3_pkgversion}-Cython >= 0.22
 %endif
+BuildRequires:  python%{python3_pkgversion}-numpy
+BuildRequires:  python%{python3_pkgversion}-simplejson
+%if 0%{?rhel}
+BuildRequires:  python%{python3_pkgversion}-yaml
+%endif
+BuildRequires:  python%{python3_pkgversion}-dill
 
-
-%description
+%global _description %{expand:
 This package is constructed on top of the MPI-1/MPI-2 specification and
 provides an object oriented interface which closely follows MPI-2 C++
 bindings. It supports point-to-point (sends, receives) and collective
 (broadcasts, scatters, gathers) communications of any picklable Python
 object as well as optimized communications of Python object exposing the
 single-segment buffer interface (NumPy arrays, built-in bytes/string/array
-objects).
+objects).}
+
+%description %_description
 
 %if 0%{?with_python2}
 %package -n python2-mpi4py
 Requires:       %{name}-common = %{version}-%{release}
 Summary:        Python 2 bindings of the Message Passing Interface (MPI)
 %{?python_provide:%python_provide python2-mpi4py}
-%description -n python2-mpi4py
-This package is constructed on top of the MPI-1/MPI-2 specification and
-provides an object oriented interface which closely follows MPI-2 C++
-bindings. It supports point-to-point (sends, receives) and collective
-(broadcasts, scatters, gathers) communications of any picklable Python
-object as well as optimized communications of Python object exposing the
-single-segment buffer interface (NumPy arrays, built-in bytes/string/array
-objects).
+%description -n python2-mpi4py %_description
```

Comment:
Existing, but do we even need to package for python2 anymore since it was EOL 4 years ago?